browser-test/src/applicant/northstar_applicant_application_statuses.test.ts

```diff
@@ -0,0 +1,58 @@
+import {test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsProgramAdmin,
+  loginAsTestUser,
+  logout,
+  testUserDisplayName,
+  validateAccessibility,
+  validateScreenshot,
+} from '../support'
+
+test.describe('with program statuses', {tag: ['@northstar']}, () => {
+  const programName = 'Applicant with statuses program'
+  const approvedStatusName = 'Approved'
+
+  test.beforeEach(
+    async ({page, adminPrograms, adminProgramStatuses, applicantQuestions}) => {
+      await loginAsAdmin(page)
+
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.gotoDraftProgramManageStatusesPage(programName)
+      await adminProgramStatuses.createStatus(approvedStatusName)
+      await adminPrograms.publishProgram(programName)
+      await adminPrograms.expectActiveProgram(programName)
+      await logout(page)
+
+      // Submit an application as a test user so that we can navigate back to the applications page.
+      await loginAsTestUser(page)
+      await applicantQuestions.clickApplyProgramButton(programName)
+      await applicantQuestions.submitFromReviewPage()
+      await logout(page)
+
+      // Navigate to the submitted application as the program admin and set a status.
+      await loginAsProgramAdmin(page)
+      await adminPrograms.viewApplications(programName)
+      await adminPrograms.viewApplicationForApplicant(testUserDisplayName())
+      const modal =
+        await adminPrograms.setStatusOptionAndAwaitModal(approvedStatusName)
+      await adminPrograms.confirmStatusUpdateModal(modal)
+      await logout(page)
+
+      await enableFeatureFlag(page, 'north_star_applicant_ui')
+    },
+  )
+
+  test.describe('applicant program index page with northstar UI', () => {
```

Comment:
**nitpick:** Since this is the northstar test file, we can probably remove "with northstar UI" here"

Reply:
I accidentally clicked "Resolve conversation" before I uploaded the change :/

Fixing this in https://github.com/civiform/civiform/pull/9147